### PR TITLE
Add type

### PR DESCRIPTION
**Pull request by JoshDunnDev**

### CODE REVIEW

File: lib/LocalVerifiableCredentialStore.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export class LocalVerifiableCredentialStore {
           'meta.profileId',
           'meta.displayable',
           'meta.parentId',
+          'content.type'
```
Comment (dlongley) [inline lib/LocalVerifiableCredentialStore.js:37]: I have all the same concerns here about indexing again. I see that the find function allows for `parentId` to be omitted and `type` to be present, for example. That would not result in the index being used properly.

What are the queries that are meant to be supported here? Which fields will be present? The indexes must match those fields. If we need another index for the queries we need to support we should add it.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Review by dlongley — APPROVED** [review]
So, I think this addresses the issue I raised before. But what happens if both `type` and `parentId` are passed? I think the caller would be confused by the outcome.

It seems like that should either still work properly -- or an exception should be thrown if it isn't supported. If it is supported, it doesn't necessarily mean a new index is needed (unless I don't understand how pouchDB works) -- you should just make it pick whichever you think would be sufficient in that situation.

If you want to defer solving this with an issue, please file one. I'm going to approve for now to keep things going. Thanks!

**Comment by JoshDunnDev** [comment]
> So, I think this addresses the issue I raised before. But what happens if both `type` and `parentId` are passed? I think the caller would be confused by the outcome.
> 
> It seems like that should either still work properly -- or an exception should be thrown if it isn't supported. If it is supported, it doesn't necessarily mean a new index is needed (unless I don't understand how pouchDB works) -- you should just make it pick whichever you think would be sufficient in that situation.
> 
> If you want to defer solving this with an issue, please file one. I'm going to approve for now to keep things going. Thanks!

@dlongley So right now there is no reason we need to call `type` and `parentId` at the same time, however this may not always be the case. I was trying to consolidate the indexing so we don't have to create a separate index for every case. We cannot include `content.type` and `meta.parentId` together because not every record includes `meta.parentId`. Since every record does not have `meta.parentId`, trying to index them together causes `meta.parentId` to be an in-memory field. I am not really sure how to get around this effectively when calling those together.

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

**Comment by dlongley** [comment]
@JoshDunnDev,

My point was that if calling the function with both `type` and `parentId` is illegal -- we should check for that condition and throw an error. Otherwise, someone may call it with both and just get back something that doesn't match `parentId` and be confused.

**Pull request merged**